DataProcessing/datax-host/src/main/scala/datax/fs/HadoopClient.scala

```diff
@@ -122,13 +122,21 @@ object HadoopClient {
     * @param sa name of the storage account
     */
   private def resolveStorageAccount(vaultName: String, sa: String) = {
+    // Fetch secret from keyvault using KeyVaultMsiAuthenticatorClient and if that does not return secret then fetch it using secret scope  
     val secretId = s"keyvault://$vaultName/${ProductConstant.ProductRoot}-sa-$sa"
```

Comment:
Who creates this secret ${ProductConstant.ProductRoot}-sa-$sa? Do we have a corresponding service change to create it? 

For now, in DataX.Flow, we currently create this kind of secret for two cases- 1. when the input type is a blob 2. when a blob output is used. Is this good enough?

Reply:
This secret is already being created by flow service. No further change is required in service